api/envoy/config/filter/http/aws_lambda/v2alpha/aws_lambda.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.aws_lambda.v2alpha;
+
+import "udpa/annotations/status.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.http.aws_lambda.v2alpha";
+option java_outer_classname = "AwsLambdaProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.filters.http.aws_lambda.v3";
+option (udpa.annotations.file_status).work_in_progress = true;
+
+// [#protodoc-title: AwsLambda]
+// AwsLambda :ref:`configuration overview <config_http_filters_aws_lambda>`.
+// [#extension: envoy.filters.http.aws_lambda]
+
+// AWS Lambda filter config
+message Config {
+  // The ARN of the AWS Lambda to invoke when the filter is engaged
+  string arn = 1;
+
+  // Whether to transform the request (headers and body) to a JSON payload or pass it as is.
+  bool payload_passthrough = 2;
+}
+
+// Per-route configuration for AWS Lambda. This can be useful when invoking a different Lambda function or a different
+// version of the same Lambda depending on the route.
+message PerRouteConfig {
```

Comment:
s/PerRouteConfig/LambdaRouteConfig/ perhaps?

Reply:
I'm going to wait on this one to see what they think.
I know there's at least one filter that does it this way. It's namespaced that's why.